net/isc-dhcp/files/dhcpd.init

```diff
@@ -517,82 +610,7 @@ general_config() {
 	max_lease_time="$(time2seconds "$max_lease_time")"
 	[ $? -ne 0 ] && return 1
 
-	if [ $dynamicdns -eq 1 ]; then
-		create_empty_zone "$domain"
-
-		local mynet
-
-		for mynet in $rfc1918_nets; do
-			mynet="$(rev_str "$mynet" ".")"
-			create_empty_zone "$mynet.in-addr.arpa"
-		done
-
-		local need_reload=
-
-		cp -p $conf_local_file ${conf_local_file}_
-
-		cat <<EOF > $conf_local_file
-zone "$domain" {
-	type master;
-	file "$dyndir/db.$domain";
-	update-policy {
-		grant $session_key_name zonesub any;
-	};
-};
-
-EOF
-
-		for mynet in $rfc1918_nets; do
```

Comment:
This logic is going away but I don't see it being replaced with anything.  How is this being handled now instead?  And can you attach a sample input config and the resultant output files into the comments?

Reply:
@jempatel ?